pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	v1alpha12 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
```

Comment:
I am confused, this doesn't seem to be used anywhre? Also please rename to `cdiv1alpha1` or something the like

Reply:
How come? it is used: https://github.com/kubermatic/machine-controller/blob/3235-machinecontroller-kubevirt-provider-allow-using-pvs-as-backing-store/pkg/cloudprovider/provider/kubevirt/provider.go#L347-L371

Reply:
ah okay, then just rename, the current import alias is really bad :P